packages/server/src/fhir/operations/utils/parameters.ts

```diff
@@ -145,26 +153,26 @@ function validateInputParam(param: OperationDefinitionParameter, value: any): an
 function parseParams(
   params: OperationDefinitionParameter[],
   inputParameters: ParametersParameter[]
-): Record<string, any> {
-  const parsed: Record<string, any> = Object.create(null);
+): Record<string, unknown> {
+  const parsed: Record<string, unknown> = Object.create(null);
   for (const param of params) {
     // FHIR spec-compliant case: Parameters resource e.g.
     // { resourceType: 'Parameters', parameter: [{ name: 'message', valueString: 'Hello!' }] }
     const inParams = inputParameters.filter((p) => p.name === param.name);
-    let value: any;
+    let value: unknown;
     if (param.part?.length) {
       value = inParams.map((input) => parseParams(param.part as [], input.part ?? []));
     } else {
       value = inParams?.map((v) => v[('value' + capitalize(param.type ?? 'string')) as keyof ParametersParameter]);
     }
 
-    parsed[param.name as string] = validateInputParam(param, value);
+    parsed[param.name] = validateInputParam(param, value);
   }
 
   return parsed;
 }
 
-export function buildOutputParameters(operation: OperationDefinition, output: any): Parameters {
+export function buildOutputParameters(operation: OperationDefinition, output: object | undefined): Parameters {
```

Comment:
Should this be `Record<string, unknown>` here instead of `object`, to avoid the cast below?

Reply:
So, yes, I would have loved that.  Unfortunately TypeScript doesn't allow you to pass normal `interface` objects as `Record<string, unknown>`, because:

```
Conversion of type 'ConceptMapTranslateOutput' to type 'Record<string, unknown>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Index signature for type 'string' is missing in type 'ConceptMapTranslateOutput'.
```
  
I find that frustrating and silly, but it is what it is?

If we change it to `Record<string, unknown>`, there would be ~50 casts required at various call sites for `as Record<string, unknown>`.

Definitely open to feedback or alternatives.